README.md

```diff
@@ -2,55 +2,24 @@
 
 # Project Name
 
-> Description the project.
+> First microverse project
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html and CSS
 
```

Comment:
[Optional] Please make your commits descriptive i.e they describe the action that you did and let them be in present tense like your second commit 

![image](https://user-images.githubusercontent.com/74744787/158657618-d23d5de8-3ec8-4621-b5eb-a1a90a058806.png)